packages/site/src/pages/GettingStarted.js

```diff
@@ -0,0 +1,149 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+import React from 'react';
+import { createStyledComponent } from '@mineral-ui/component-utils';
+import CodeBlock from '../components/CodeBlock';
+import Link from '../components/Link';
+import PageLayout from '../components/PageLayout';
+
+const Callout = createStyledComponent('p', ({ theme }) => ({
+  backgroundColor: theme.color_gray_10,
+  borderLeft: `4px solid ${theme.borderColor}`,
+  padding: `${theme.spacing_single} ${theme.spacing_double}`
+}));
+
+export default function GettingStarted() {
+  return (
```

Comment:
The content of this getting started page is going to change substantially in the next 2 weeks due to the architecture updates.  We should probably hold off on including this content, in order to avoid the need to simply update it again.

Reply:
It’ll just be content changes, which shouldn’t take long to update and neither of us mind doing so. Additionally, we’d rather not merge this website work (on `web-mvp` branch) without a Getting Started page, as it would just be Guidelines and Theming without it.